py/desisim/scripts/quickquasars.py

```diff
@@ -618,6 +678,14 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
 
         tmp_qso_flux = apply_metals_transmission(tmp_qso_wave,tmp_qso_flux,
                             trans_wave,transmission,args.metals)
+    # Attenuate the spectra for extinction
+    if not sfdmap is None:
+        Rv=3.1   #set by default
+        indx=np.arange(metadata['RA'].size)
+        extinction =Rv*ext_odonnell(tmp_qso_wave)
+        EBV = sfdmap.ebv(metadata['RA'],metadata['DEC'], scaling=1.0)
+        tmp_qso_flux *=10**( -0.4 * EBV[indx, np.newaxis] * extinction)
+        log.info("Dust extinction added")
```

Comment:
I don't know the details about this extinsion corrections, but I guess you have cross-checked this works?